### PR TITLE
DOCS-6267 - ADW 2.2 Media Management - Image manipulation and IPTC metadata extract

### PR DESCRIPTION
**Pull request by gravitonian**

### CODE REVIEW

File: _data/toc/digital-workspace.yaml [hunk 1]
```diff
@@ -35,6 +35,8 @@
           path: '/digital-workspace/latest/process/tasks/'
         - title: 'Configuring Process Services'
           path: '/digital-workspace/latest/process/config/'
+    - title: 'Media Management'
```
Comment (pnirwal) [inline _data/toc/digital-workspace.yaml:38]: Please change all references to "Media Management" to "Image Management"

File: digital-workspace/latest/media/index.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+title: Image Management
+---
+
+Image Management allows you to view and manipulate your image files in the Digital Workspace.
+
+Features provided with Image Management include enhanced image manipulation with image crop and rotate capabilities
+and [IPTC](https://iptc.org/standards/photo-metadata/){:target="_blank"} metadata extraction. 
+
+## Uploading images
+Image Management provides information and features about image files that you upload in Digital Workspace.
```
Comment (oliveralfresco2018) [inline digital-workspace/latest/media/index.md:11]:
```suggestion
Image Management provides information and features about image files that you upload in the Digital Workspace.
```

File: digital-workspace/latest/media/index.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+1. Select the folder in the File Library where you want to add your content.
+
+2. You can drag and drop images, or select **NEW -> Upload File** from the toolbar, as you would normally do in Digital Workspace.
```
Comment (oliveralfresco2018) [inline digital-workspace/latest/media/index.md:15]:
```suggestion
2. You can drag and drop images, or select **NEW -> Upload File** from the toolbar, as you would normally in the Digital Workspace.
```

File: digital-workspace/latest/media/index.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+   ![image-preview]({% link digital-workspace/images/adw-media-props-sections.png %}){:height="300px" width="150px"}
+
+5. Click the **IPTC** section to see IPTC metadata:
+
+   ![image-preview]({% link digital-workspace/images/adw-iptc-metadata.png %}){:height="600px" width="150px"}
+
+>**Note:** IPTC metadata is also mapped to existing description (IPTC caption/description) and title (IPTC headline) properties, hover over the filename in the File library, and the IPTC property values are displayed.
+   
+>**Note**. The IPTC Content Model needs to be bootstrapped into Alfresco Content Services for the IPTC metadata extraction to work.
```
Comment (oliveralfresco2018) [inline digital-workspace/latest/media/index.md:48]:
```suggestion
>**Note:** The IPTC Content Model needs to be bootstrapped into Alfresco Content Services for the IPTC metadata extraction to work.
```

File: digital-workspace/latest/media/index.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+ 
+## Manipulating images
+Image Management provides features to allow you to edit image files in Digital Workspace.
+
+1. Select an image from the File Library, as you would normally in Digital Workspace, by clicking the thumbnail or name, to view it in the file preview screen.
```
Comment (oliveralfresco2018) [inline digital-workspace/latest/media/index.md:53]:
```suggestion
1. Select an image from the File Library, as you would normally in the Digital Workspace, by clicking the thumbnail or name, to view it in the file preview screen.
```

**Review by oliveralfresco2018 — COMMENTED** [review]
minor comments, hope you're feeling better :-)

**Pull request merged**